package.json

```diff
@@ -97,21 +98,20 @@
   "scripts": {
     "lint": "eslint lib test",
     "test": "if [ $COVERAGE ]; then npm run coveralls; else npm run jshint && npm run teaser && npm run test-unit && npm run test-integration; fi",
-    "test-docker": "docker-compose run sequelize /bin/sh -c \"npm run test-all\"",
-    "test-docker-unit": "docker-compose run sequelize /bin/sh -c \"npm run test-unit-all\"",
-    "test-docker-integration": "docker-compose run sequelize /bin/sh -c \"npm run test-integration-all\"",
-    "build-docker": "docker-compose build",
+    "test-docker": "env-cmd ./test/config/.docker.env npm run test-all",
+    "test-docker-unit": "env-cmd ./test/config/.docker.env npm run test-unit-all",
+    "test-docker-integration": "env-cmd ./test/config/.docker.env npm run test-integration-all",
     "docs": "node docs/docs-generator.js",
-    "jshint": "./node_modules/.bin/jshint lib test",
```

Comment:
See #5963 #6288 #6436 
We are switching to ESLint but there are many lint issues JSHint didn't detect. So in CI JSHint is still used but we have the eslintrc for developers and their IDEs.